webrtc.html

```diff
@@ -987,6 +987,64 @@
             yet been set.</p>
           </dd>
 
+          <dt>Promise&lt;void&gt; addIceCandidate (RTCIceCandidate candidate)</dt>
+
+          <dd>
+            <p>The <dfn id=
+            "dom-peerconnection-addicecandidate"><code>addIceCandidate()</code></dfn>
+            method provides a remote candidate to the ICE Agent. In addition to
+            being added to the remote description, connectivity checks will be
+            sent to the new candidates as long as the <a href=
+            "#ice-transports-setting">ICE Transports setting</a> is not set to
+            <code>none</code>. This call will result in a change to the
+            connection state of the ICE Agent, and may result in a change to
+            media state if it results in different connectivity being
+            established.</p>
+
+            <ol>
+              <li>
+                <p>Let <var>p</var> be a new promise.</p>
+              </li>
+
+              <li>
+                <p>If this <code><a>RTCPeerConnection</a></code> object's
+                <a href="#dom-peerconnection-signaling-state">signaling
+                state</a> is <code>closed</code>, the user agent MUST reject
+                <var>p</var> with <code>InvalidStateError</code>, and
+                jump to the step labeled <em>Return</em>.</p>
+              </li>
+
+              <li>
+                <p>If the candidate parameter is malformed, reject <var>p</var>
+                with <code>SyntaxError</code> and jump to the step labeled
+                <em>Return</em>.</p>
+              </li>
+
+              <li>
+                <p>If the candidate could not be successfully applied, reject
+                <var>p</var> with a <code>DOMError</code> object whose
+                <code>name</code> attribute has the value TBD (TODO
+                InvalidCandidate and InvalidMidIndex) and jump to the step
+                labeled <em>Return</em>.</p>
```

Comment:
It seems like this could be a little more extensively defined than "successfully applied", even if it is just a reference to JSEP.